api/rest/v1/bulk_info_place.md

```diff
@@ -12,7 +12,7 @@ permalink: /api/rest/v1/bulk/info/place
 
 ## /v1/bulk/info/place
 
-Get basic information about multiple [places](/api/rest/v1/getting_started#place).
+Get basic information about multiple [places](/glossary.md#place).
```

Comment:
change all these to glossary.html instead of .md

Reply:
Done